mappings/net/minecraft/entity/EntityPosition.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_10182 net/minecraft/entity/player/PlayerPosition
+CLASS net/minecraft/class_10182 net/minecraft/entity/EntityPosition
```

Comment:
Probably going to look at this more closely later, but I just wanted to note that there's an `EntityPos` interface since last snapshot (which maybe should've been in `entity` instead of `util`…)

Reply:
Ah, I didn't see that, that sounds good